processor/spanbatchsamplerprocessor/testdata/config.yaml

```diff
@@ -0,0 +1 @@
+tokens: 1000
```

Comment:
The config on the testdata should be a complete config for the component, potentially with different component IDs ("spanbatchsampler/1", "spanbatchsampler/default", ...)